WordPress/Classes/ViewRelated/Comments/Content/CommentContentHeaderView.swift

```diff
@@ -0,0 +1,197 @@
+import SwiftUI
+import DesignSystem
+
+struct CommentContentHeaderView: View {
+
+    let config: Configuration
+
+    var body: some View {
+        HStack(spacing: .DS.Padding.split) {
+            AvatarsView(style: .single(config.avatarURL))
+            VStack(alignment: .leading) {
+                Text(config.username)
+                    .style(.bodySmall(.regular))
+                    .foregroundStyle(Color.DS.Foreground.primary)
+                    .lineLimit(1)
+                Text(config.handleAndTimestamp)
+                    .style(.bodySmall(.regular))
+                    .foregroundStyle(Color.DS.Foreground.secondary)
+                    .lineLimit(1)
+            }
+            Spacer()
+            CommentContentHeaderMenu(menu: config.menu)
+        }
+    }
+
+    struct Configuration {
+        let avatarURL: URL?
+        let username: String
+        let handleAndTimestamp: String
+        let menu: [[MenuItem]]
```

Comment:
I think it isn't very obvious why menu is an array of arrays. Could we improve the data structure to make the use case more explicit? Or perhaps rename it to highlight the sections & rows structure?

Reply:
Resolved in c5ba22dea8ed983d4264bdf378a1b98e1680d6f9